### ISSUE TITLE
Why Pelee?

### ISSUE THREAD

**Issue by talenz** [Robert-JunWang/Pelee#56]
What's the meaning of Pelee? Abbreviation for ?

**Comment by Robert-JunWang** [comment]
No. I would be grateful if you could help with the meaning of the appropriate acronym. I am not very good at naming. The reason why this model is called Pelee is mainly that I was on vacation at Point Pelee Park when I started this job. Another reason is that Pelee sounds very much like Chinese lightning, which is often used in Chinese to describe the fast speed. Fast is exactly the main goal that this model hopes to achieve.

**Comment by talenz** [comment]
霹雳！Great!




            
            
                
                    
                            
                                
                            
                            
                                tlzjay
                            
                    
                        
                            
                                    邮箱：tlzjay@gmail.com
                            
                        
                
            
        Signature is customized by Netease Mail Master


On 12/29/2018 11:52, Robert J. Wang wrote: No. I would be grateful if you could help with the meaning of the appropriate acronym. I am not very good at naming. The reason why this model is called Pelee is mainly that I was on vacation at Point Pelee Park when I started this job. Another reason is that Pelee sounds very much like Chinese lightning, which is often used in Chinese to describe the fast speed. Fast is exactly the main goal that this model hopes to achieve.

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Robert-JunWang/Pelee","title":"Robert-JunWang/Pelee","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/Robert-JunWang/Pelee"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Robert-JunWang in #56: No. I would be grateful if you could help with the meaning of the appropriate acronym. I am not very good at naming. The reason why this model is called Pelee is mainly that I was on vacation at Point Pelee Park when I started this job. Another reason is that Pelee sounds very much like Chinese lightning, which is often used in Chinese to describe the fast speed. Fast is exactly the main goal that this model hopes to achieve."}],"action":{"name":"View Issue","url":"https://github.com/Robert-JunWang/Pelee/issues/56#issuecomment-450463086"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/Robert-JunWang/Pelee/issues/56#issuecomment-450463086",
"url": "https://github.com/Robert-JunWang/Pelee/issues/56#issuecomment-450463086",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by gaobb** [comment]
> No. I would be grateful if you could help with the meaning of the appropriate acronym. I am not very good at naming. The reason why this model is called Pelee is mainly that I was on vacation at Point Pelee Park when I started this job. Another reason is that Pelee sounds very much like Chinese lightning, which is often used in Chinese to describe the fast speed. Fast is exactly the main goal that this model hopes to achieve.

very interesting!

**Comment by nullbyte91** [comment]
> No. I would be grateful if you could help with the meaning of the appropriate acronym. I am not very good at naming. The reason why this model is called Pelee is mainly that I was on vacation at Point Pelee Park when I started this job. Another reason is that Pelee sounds very much like Chinese lightning, which is often used in Chinese to describe the fast speed. Fast is exactly the main goal that this model hopes to achieve.

Great.
